utils/setup/install_deps.sh

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env bash
 
 function check_python_version_gte_3_7 {
-    echo "Checking for >=python3.7"
+    echo "Checking for >=python3.8"
     # running through current minor versions
-    hash python3.7 2>/dev/null \
+    hash python3.8 2>/dev/null \
     || hash python3.8 2>/dev/null \
     || hash python3.9 2>/dev/null;
 }
 
 function install_python_3_7 {
-    echo "Installing python3.7"
+    echo "Installing python3.8"
```

Comment:
Maybe the function names can also be update, such as `function install_python_3_8` .